data/templates/template.typst

```diff
@@ -35,55 +35,50 @@
     paper: paper,
     margin: margin,
     numbering: "1",
-  )
+    columns: cols,
+    )
+
   set par(justify: true)
   set text(lang: lang,
            region: region,
            font: font,
            size: fontsize)
   set heading(numbering: sectionnumbering)
 
-  if title != none {
-    align(center)[#block(inset: 2em)[
-      #text(weight: "bold", size: 1.5em)[#title]
-      #(if subtitle != none {
+  place(top + center, float: true, scope: "parent", clearance: 4mm)[
+    #if title != none {
+      text(weight: "bold", size: 1.5em)[#title]
+      (if subtitle != none {
         parbreak()
         text(weight: "bold", size: 1.25em)[#subtitle]
       })
-    ]]
-  }
-
-  if authors != none and authors != [] {
-    let count = authors.len()
-    let ncols = calc.min(count, 3)
-    grid(
-      columns: (1fr,) * ncols,
-      row-gutter: 1.5em,
-      ..authors.map(author =>
-          align(center)[
-            #author.name \
-            #author.affiliation \
-            #author.email
-          ]
-      )
-    )
-  }
+    }
 
-  if date != none {
-    align(center)[#block(inset: 1em)[
-      #date
-    ]]
-  }
+    #if authors != none and authors != [] {
+      let count = authors.len()
+      let ncols = calc.min(count, 3)
+       grid(
+        columns: (1fr,) * ncols,
+        row-gutter: 1.5em,
+        ..authors.map(author =>
+            align(center)[
+              #author.name \
+              #author.affiliation \
+              #author.email
+            ]
+        )
+       )
+    }
 
-  if abstract != none {
-    block(inset: 2em)[
-    #text(weight: "semibold")[Abstract] #h(1em) #abstract
-    ]
-  }
+    #if date != none {
+        date
+    }
 
-  if cols == 1 {
-    doc
-  } else {
-    columns(cols, doc)
-  }
+    #if abstract != none {
+      text(weight: "semibold")[Abstract]
+      h(1em) 
+      abstract
```

Comment:
Can you confirm that this gives the same visual appearance as the previous template?  Before we had a block with 2em inset.  The `#block` has been removed.  What effect does that have?

Reply:
I believe it's all cleaned up now...